RACIPELIB.c

```diff
@@ -232,7 +232,7 @@ void check_inputfile (int argc, char **argv, struct topo *topoinfo, struct opts
     printf("-thrd          : Cutoff for convergence of steady states for numerically solving ODEs (Default 1.0).\n");
     printf("-Toggle_f_p    : Save parameters of each RACIPE model or not (Default 1 (yes)).\n");
     printf("-stepsize      : Stepsize for solving ODEs (Default 0.1).\n");
-    printf("-maxiters      : The maximum number of iterations for solving ODEs at each random initial condition (Default 20).\n");
+    printf("-maxiters      : The maximum number of iterations for solving ODEs at each random initial condition times 1000 (Default 20).\n");
```

Comment:
Why would you add "times 100" here? Bc of RIV?

Reply:
The main aim was to have `20` replaced by `simu_opts->maxiters`.  The rest was only a suggested correction.

In both the `euler` and `rk45` solvers, if I understand it correctly, the actual differential equation updating iteration (`y_new = y_old + delta_y * stepsize`) occurs as a part of a double loop, with the inner loop run from 1 to `n_step = 1000` every time, while the outer loop depends on the `maxiters` option. So overall, the total number of times this update step occurs is 1000 * maxiters, which is the maximum iterations of the update step. Hence changed the wordings a little bit.

In case I misunderstood the workings of the code, or the intention of the documentation, remove this from the pull request.